docker/docker-darwin.go

```diff
@@ -2,47 +2,18 @@
 
 package docker
 
-import (
-	"fmt"
-	"os"
-	"os/exec"
-	"strconv"
-	"strings"
-)
+func (r *Runner) CheckProjectDir() error {
+	return nil
+}
 
-var (
-	errDockerMachineUnknown = fmt.Errorf("unknown DOCKER_MACHINE_NAME")
-)
+func (r *Runner) ProjectDir() string{
+	return r.projectDir
+}
```

Comment:
Can we make `CheckProjectDir` and `ProjectDir` unexported? I don't think they will be used outside docker package.